### DISCUSSION TITLE
Minimun Laravel Version currently? 2020

### DISCUSSION THREAD

**Discussion by amoralesdesign** [livewire/livewire#2200] [1. Help]
My PHP version (7.2.31)
And my Laravel version  "laravel/framework": "5.7.*",

I try to install "livewire/livewire": "^2.3"

But get this error in terminal:
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - illuminate/validation v8.9.0 requires php ^7.3 -> your PHP version (7.2.31) does not satisfy that requirement.
    - illuminate/validation v8.8.0 requires php ^7.3 -> your PHP version (7.2.31) does not satisfy that requirement.
    - illuminate/validation v8.7.1 requires php ^7.3 -> your PHP version (7.2.31) does not satisfy that requirement.
    - illuminate/validation v8.7.0 requires php ^7.3 -> your PHP version (7.2.31) does not satisfy that requirement.
    - illuminate/validation v8.6.0 requires php ^7.3 -> your PHP version (7.2.31) does not satisfy that requirement.
    - illuminate/validation v8.5.0 requires php ^7.3 -> your PHP version (7.2.31) does not satisfy that requirement.
    - illuminate/validation v8.4.0 requires php ^7.3 -> your PHP version (7.2.31) does not satisfy that requirement.
    - illuminate/validation v8.3.0 requires php ^7.3 -> your PHP version (7.2.31) does not satisfy that requirement.
    - illuminate/validation v8.2.0 requires php ^7.3 -> your PHP version (7.2.31) does not satisfy that requirement.
    - illuminate/validation v8.18.1 requires php ^7.3|^8.0 -> your PHP version (7.2.31) does not satisfy that requirement.
    - illuminate/validation v8.18.0 requires php ^7.3|^8.0 -> your PHP version (7.2.31) does not satisfy that requirement.
    - illuminate/validation v8.17.2 requires php ^7.3|^8.0 -> your PHP version (7.2.31) does not satisfy that requirement.
    - illuminate/validation v8.17.0 requires php ^7.3|^8.0 -> your PHP version (7.2.31) does not satisfy that requirement.
    - illuminate/validation v8.16.1 requires php ^7.3|^8.0 -> your PHP version (7.2.31) does not satisfy that requirement.
    - illuminate/validation v8.16.0 requires php ^7.3|^8.0 -> your PHP version (7.2.31) does not satisfy that requirement.
    - illuminate/validation v8.15.0 requires php ^7.3|^8.0 -> your PHP version (7.2.31) does not satisfy that requirement.
    - illuminate/validation v8.14.0 requires php ^7.3|^8.0 -> your PHP version (7.2.31) does not satisfy that requirement.
    - illuminate/validation v8.13.0 requires php ^7.3|^8.0 -> your PHP version (7.2.31) does not satisfy that requirement.
    - illuminate/validation v8.12.3 requires php ^7.3|^8.0 -> your PHP version (7.2.31) does not satisfy that requirement.
    - illuminate/validation v8.12.2 requires php ^7.3|^8.0 -> your PHP version (7.2.31) does not satisfy that requirement.
    - illuminate/validation v8.12.1 requires php ^7.3|^8.0 -> your PHP version (7.2.31) does not satisfy that requirement.
    - illuminate/validation v8.12.0 requires php ^7.3|^8.0 -> your PHP version (7.2.31) does not satisfy that requirement.
    - illuminate/validation v8.11.2 requires php ^7.3 -> your PHP version (7.2.31) does not satisfy that requirement.
    - illuminate/validation v8.11.1 requires php ^7.3 -> your PHP version (7.2.31) does not satisfy that requirement.
    - illuminate/validation v8.11.0 requires php ^7.3 -> your PHP version (7.2.31) does not satisfy that requirement.
    - illuminate/validation v8.10.0 requires php ^7.3 -> your PHP version (7.2.31) does not satisfy that requirement.
    - illuminate/validation v8.1.0 requires php ^7.3 -> your PHP version (7.2.31) does not satisfy that requirement.
    - illuminate/validation v8.0.4 requires php ^7.3 -> your PHP version (7.2.31) does not satisfy that requirement.
    - illuminate/validation v8.0.3 requires php ^7.3 -> your PHP version (7.2.31) does not satisfy that requirement.
    - illuminate/validation v8.0.2 requires php ^7.3 -> your PHP version (7.2.31) does not satisfy that requirement.
    - illuminate/validation v8.0.1 requires php ^7.3 -> your PHP version (7.2.31) does not satisfy that requirement.
    - illuminate/validation v8.0.0 requires php ^7.3 -> your PHP version (7.2.31) does not satisfy that requirement.
    - illuminate/validation 8.x-dev requires php ^7.3|^8.0 -> your PHP version (7.2.31) does not satisfy that requirement.
    - Conclusion: remove laravel/framework v5.7.28
    - Conclusion: don't install laravel/framework v5.7.28
    - livewire/livewire v2.3.0 requires illuminate/validation ^7.0|^8.0 -> satisfiable by illuminate/validation[7.x-dev, 8.x-dev, v7.0.0, v7.0.1, v7.0.2, v7.0.3, v7.0.4, v7.0.5, v7.0.6, v7.0.7, v7.0.8, v7.1.0, v7.1.1, v7.1.2, v7.1.3, v7.10.0, v7.10.1, v7.10.2, v7.10.3, v7.11.0, v7.12.0, v7.13.0, v7.14.0, v7.14.1, v7.15.0, v7.16.0, v7.16.1, v7.17.0, v7.17.1, v7.17.2, v7.18.0, v7.19.0, v7.19.1, v7.2.0, v7.2.1, v7.2.2, v7.20.0, v7.21.0, v7.22.0, v7.22.1, v7.22.2, v7.22.3, v7.22.4, v7.23.0, v7.23.1, v7.23.2, v7.24.0, v7.25.0, v7.26.0, v7.26.1, v7.27.0, v7.28.0, v7.28.1, v7.28.2, v7.28.3, v7.28.4, v7.29.0, v7.29.1, v7.29.2, v7.29.3, v7.3.0, v7.30.0, v7.4.0, v7.5.0, v7.5.1, v7.5.2, v7.6.0, v7.6.1, v7.6.2, v7.7.0, v7.7.1, v7.8.0, v7.8.1, v7.9.0, v7.9.1, v7.9.2, v8.0.0, v8.0.1, v8.0.2, v8.0.3, v8.0.4, v8.1.0, v8.10.0, v8.11.0, v8.11.1, v8.11.2, v8.12.0, v8.12.1, v8.12.2, v8.12.3, v8.13.0, v8.14.0, v8.15.0, v8.16.0, v8.16.1, v8.17.0, v8.17.2, v8.18.0, v8.18.1, v8.2.0, v8.3.0, v8.4.0, v8.5.0, v8.6.0, v8.7.0, v8.7.1, v8.8.0, v8.9.0].
    - livewire/livewire v2.3.1 requires illuminate/validation ^7.0|^8.0 -> satisfiable by illuminate/validation[7.x-dev, 8.x-dev, v7.0.0, v7.0.1, v7.0.2, v7.0.3, v7.0.4, v7.0.5, v7.0.6, v7.0.7, v7.0.8, v7.1.0, v7.1.1, v7.1.2, v7.1.3, v7.10.0, v7.10.1, v7.10.2, v7.10.3, v7.11.0, v7.12.0, v7.13.0, v7.14.0, v7.14.1, v7.15.0, v7.16.0, v7.16.1, v7.17.0, v7.17.1, v7.17.2, v7.18.0, v7.19.0, v7.19.1, v7.2.0, v7.2.1, v7.2.2, v7.20.0, v7.21.0, v7.22.0, v7.22.1, v7.22.2, v7.22.3, v7.22.4, v7.23.0, v7.23.1, v7.23.2, v7.24.0, v7.25.0, v7.26.0, v7.26.1, v7.27.0, v7.28.0, v7.28.1, v7.28.2, v7.28.3, v7.28.4, v7.29.0, v7.29.1, v7.29.2, v7.29.3, v7.3.0, v7.30.0, v7.4.0, v7.5.0, v7.5.1, v7.5.2, v7.6.0, v7.6.1, v7.6.2, v7.7.0, v7.7.1, v7.8.0, v7.8.1, v7.9.0, v7.9.1, v7.9.2, v8.0.0, v8.0.1, v8.0.2, v8.0.3, v8.0.4, v8.1.0, v8.10.0, v8.11.0, v8.11.1, v8.11.2, v8.12.0, v8.12.1, v8.12.2, v8.12.3, v8.13.0, v8.14.0, v8.15.0, v8.16.0, v8.16.1, v8.17.0, v8.17.2, v8.18.0, v8.18.1, v8.2.0, v8.3.0, v8.4.0, v8.5.0, v8.6.0, v8.7.0, v8.7.1, v8.8.0, v8.9.0].
    - livewire/livewire v2.3.2 requires illuminate/validation ^7.0|^8.0 -> satisfiable by illuminate/validation[7.x-dev, 8.x-dev, v7.0.0, v7.0.1, v7.0.2, v7.0.3, v7.0.4, v7.0.5, v7.0.6, v7.0.7, v7.0.8, v7.1.0, v7.1.1, v7.1.2, v7.1.3, v7.10.0, v7.10.1, v7.10.2, v7.10.3, v7.11.0, v7.12.0, v7.13.0, v7.14.0, v7.14.1, v7.15.0, v7.16.0, v7.16.1, v7.17.0, v7.17.1, v7.17.2, v7.18.0, v7.19.0, v7.19.1, v7.2.0, v7.2.1, v7.2.2, v7.20.0, v7.21.0, v7.22.0, v7.22.1, v7.22.2, v7.22.3, v7.22.4, v7.23.0, v7.23.1, v7.23.2, v7.24.0, v7.25.0, v7.26.0, v7.26.1, v7.27.0, v7.28.0, v7.28.1, v7.28.2, v7.28.3, v7.28.4, v7.29.0, v7.29.1, v7.29.2, v7.29.3, v7.3.0, v7.30.0, v7.4.0, v7.5.0, v7.5.1, v7.5.2, v7.6.0, v7.6.1, v7.6.2, v7.7.0, v7.7.1, v7.8.0, v7.8.1, v7.9.0, v7.9.1, v7.9.2, v8.0.0, v8.0.1, v8.0.2, v8.0.3, v8.0.4, v8.1.0, v8.10.0, v8.11.0, v8.11.1, v8.11.2, v8.12.0, v8.12.1, v8.12.2, v8.12.3, v8.13.0, v8.14.0, v8.15.0, v8.16.0, v8.16.1, v8.17.0, v8.17.2, v8.18.0, v8.18.1, v8.2.0, v8.3.0, v8.4.0, v8.5.0, v8.6.0, v8.7.0, v8.7.1, v8.8.0, v8.9.0].
    - livewire/livewire v2.3.3 requires illuminate/validation ^7.0|^8.0 -> satisfiable by illuminate/validation[7.x-dev, 8.x-dev, v7.0.0, v7.0.1, v7.0.2, v7.0.3, v7.0.4, v7.0.5, v7.0.6, v7.0.7, v7.0.8, v7.1.0, v7.1.1, v7.1.2, v7.1.3, v7.10.0, v7.10.1, v7.10.2, v7.10.3, v7.11.0, v7.12.0, v7.13.0, v7.14.0, v7.14.1, v7.15.0, v7.16.0, v7.16.1, v7.17.0, v7.17.1, v7.17.2, v7.18.0, v7.19.0, v7.19.1, v7.2.0, v7.2.1, v7.2.2, v7.20.0, v7.21.0, v7.22.0, v7.22.1, v7.22.2, v7.22.3, v7.22.4, v7.23.0, v7.23.1, v7.23.2, v7.24.0, v7.25.0, v7.26.0, v7.26.1, v7.27.0, v7.28.0, v7.28.1, v7.28.2, v7.28.3, v7.28.4, v7.29.0, v7.29.1, v7.29.2, v7.29.3, v7.3.0, v7.30.0, v7.4.0, v7.5.0, v7.5.1, v7.5.2, v7.6.0, v7.6.1, v7.6.2, v7.7.0, v7.7.1, v7.8.0, v7.8.1, v7.9.0, v7.9.1, v7.9.2, v8.0.0, v8.0.1, v8.0.2, v8.0.3, v8.0.4, v8.1.0, v8.10.0, v8.11.0, v8.11.1, v8.11.2, v8.12.0, v8.12.1, v8.12.2, v8.12.3, v8.13.0, v8.14.0, v8.15.0, v8.16.0, v8.16.1, v8.17.0, v8.17.2, v8.18.0, v8.18.1, v8.2.0, v8.3.0, v8.4.0, v8.5.0, v8.6.0, v8.7.0, v8.7.1, v8.8.0, v8.9.0].
    - livewire/livewire v2.3.4 requires illuminate/validation ^7.0|^8.0 -> satisfiable by illuminate/validation[7.x-dev, 8.x-dev, v7.0.0, v7.0.1, v7.0.2, v7.0.3, v7.0.4, v7.0.5, v7.0.6, v7.0.7, v7.0.8, v7.1.0, v7.1.1, v7.1.2, v7.1.3, v7.10.0, v7.10.1, v7.10.2, v7.10.3, v7.11.0, v7.12.0, v7.13.0, v7.14.0, v7.14.1, v7.15.0, v7.16.0, v7.16.1, v7.17.0, v7.17.1, v7.17.2, v7.18.0, v7.19.0, v7.19.1, v7.2.0, v7.2.1, v7.2.2, v7.20.0, v7.21.0, v7.22.0, v7.22.1, v7.22.2, v7.22.3, v7.22.4, v7.23.0, v7.23.1, v7.23.2, v7.24.0, v7.25.0, v7.26.0, v7.26.1, v7.27.0, v7.28.0, v7.28.1, v7.28.2, v7.28.3, v7.28.4, v7.29.0, v7.29.1, v7.29.2, v7.29.3, v7.3.0, v7.30.0, v7.4.0, v7.5.0, v7.5.1, v7.5.2, v7.6.0, v7.6.1, v7.6.2, v7.7.0, v7.7.1, v7.8.0, v7.8.1, v7.9.0, v7.9.1, v7.9.2, v8.0.0, v8.0.1, v8.0.2, v8.0.3, v8.0.4, v8.1.0, v8.10.0, v8.11.0, v8.11.1, v8.11.2, v8.12.0, v8.12.1, v8.12.2, v8.12.3, v8.13.0, v8.14.0, v8.15.0, v8.16.0, v8.16.1, v8.17.0, v8.17.2, v8.18.0, v8.18.1, v8.2.0, v8.3.0, v8.4.0, v8.5.0, v8.6.0, v8.7.0, v8.7.1, v8.8.0, v8.9.0].
    - livewire/livewire v2.3.5 requires illuminate/validation ^7.0|^8.0 -> satisfiable by illuminate/validation[7.x-dev, 8.x-dev, v7.0.0, v7.0.1, v7.0.2, v7.0.3, v7.0.4, v7.0.5, v7.0.6, v7.0.7, v7.0.8, v7.1.0, v7.1.1, v7.1.2, v7.1.3, v7.10.0, v7.10.1, v7.10.2, v7.10.3, v7.11.0, v7.12.0, v7.13.0, v7.14.0, v7.14.1, v7.15.0, v7.16.0, v7.16.1, v7.17.0, v7.17.1, v7.17.2, v7.18.0, v7.19.0, v7.19.1, v7.2.0, v7.2.1, v7.2.2, v7.20.0, v7.21.0, v7.22.0, v7.22.1, v7.22.2, v7.22.3, v7.22.4, v7.23.0, v7.23.1, v7.23.2, v7.24.0, v7.25.0, v7.26.0, v7.26.1, v7.27.0, v7.28.0, v7.28.1, v7.28.2, v7.28.3, v7.28.4, v7.29.0, v7.29.1, v7.29.2, v7.29.3, v7.3.0, v7.30.0, v7.4.0, v7.5.0, v7.5.1, v7.5.2, v7.6.0, v7.6.1, v7.6.2, v7.7.0, v7.7.1, v7.8.0, v7.8.1, v7.9.0, v7.9.1, v7.9.2, v8.0.0, v8.0.1, v8.0.2, v8.0.3, v8.0.4, v8.1.0, v8.10.0, v8.11.0, v8.11.1, v8.11.2, v8.12.0, v8.12.1, v8.12.2, v8.12.3, v8.13.0, v8.14.0, v8.15.0, v8.16.0, v8.16.1, v8.17.0, v8.17.2, v8.18.0, v8.18.1, v8.2.0, v8.3.0, v8.4.0, v8.5.0, v8.6.0, v8.7.0, v8.7.1, v8.8.0, v8.9.0].
    - don't install illuminate/validation 7.x-dev|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.0.0|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.0.1|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.0.2|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.0.3|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.0.4|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.0.5|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.0.6|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.0.7|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.0.8|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.1.0|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.1.1|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.1.2|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.1.3|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.10.0|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.10.1|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.10.2|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.10.3|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.11.0|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.12.0|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.13.0|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.14.0|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.14.1|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.15.0|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.16.0|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.16.1|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.17.0|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.17.1|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.17.2|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.18.0|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.19.0|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.19.1|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.2.0|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.2.1|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.2.2|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.20.0|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.21.0|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.22.0|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.22.1|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.22.2|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.22.3|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.22.4|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.23.0|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.23.1|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.23.2|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.24.0|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.25.0|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.26.0|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.26.1|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.27.0|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.28.0|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.28.1|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.28.2|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.28.3|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.28.4|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.29.0|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.29.1|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.29.2|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.29.3|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.3.0|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.30.0|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.4.0|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.5.0|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.5.1|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.5.2|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.6.0|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.6.1|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.6.2|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.7.0|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.7.1|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.8.0|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.8.1|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.9.0|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.9.1|don't install laravel/framework v5.7.28
    - don't install illuminate/validation v7.9.2|don't install laravel/framework v5.7.28
    - Installation request for laravel/framework (locked at v5.7.28, required as 5.7.*) -> satisfiable by laravel/framework[v5.7.28].
    - Installation request for livewire/livewire ^2.3 -> satisfiable by livewire/livewire[v2.3.0, v2.3.1, v2.3.2, v2.3.3, v2.3.4, v2.3.5].

**Comment by joshhanley** [comment]
Just saw this post, have answered in #2201.

@amoralesdesign please only open one issue or discussion in future, not both. 
It's preferred that you open a discussion, so issues are for any confirmed bugs with Livewire.

Thanks!
